bin/utils/process-definition.sh

```diff
@@ -18,5 +18,6 @@ docker run --rm --name json2xlsx \
   -v ${definition_input_dir}:/tmp/ccd-definition \
   -v ${definition_output_file}:/tmp/ccd-definition.xlsx \
   -e CCD_DEF_CASE_SERVICE_BASE_URL=${CCD_DEF_CASE_SERVICE_BASE_URL:-http://docker.for.mac.localhost:4550} \
```

Comment:
should the port here be 4000? since CCD_DEF_CASE_SERVICE_BASE_URL points to civil-service?